testsuite/MDAnalysisTests/__init__.py

```diff
@@ -97,7 +97,7 @@
 logger = logging.getLogger("MDAnalysisTests.__init__")
 
 # keep in sync with RELEASE in setup.py
-__version__ = "2.10.0"
+__version__ = "2.11.0-dev0"
```

Comment:
```suggestion
__version__ = "2.11.0-dev0"  # NOTE: keep in sync with RELEASE in setup.py
```
For consistency? Doesn't hurt...

Reply:
Sure